templates/includes/event_hero.html

```diff
@@ -1,5 +1,5 @@
 {% load wagtailimages_tags %}
-<div class="event-hero">
+<header class="event-hero">
```

Comment:
This now means there are two `<header>` elements on the page - is that correct? I am guessing this is OK because it is wrapped in an `<article>`?

Reply:
Yeah it's fine to have more than one, and as you say this one is separated by the article tag as well which distinguishes it from the main template header